code/modules/mob/living/silicon/pai/pai_defense.dm

```diff
@@ -12,6 +12,14 @@
 	if(holoform)
 		fold_in(force = TRUE)
 	//Need more effects that aren't instadeath or permanent law corruption.
+	//With pAIs becoming more and more useful, bigger drawbacks are needed to make them easier to counter
+	switch(severity)
+		if(1) //death
+			qdel(src)
```

Comment:
this is stupid, pai just disappearing lol

Reply:
> this is stupid, pai just disappearing lol

![thanos-infinity](https://user-images.githubusercontent.com/82319489/129460687-533cfb1a-d056-44ce-a87b-349f2e03ce08.gif)

Reply:
Qdels the mob (holoform), leaves the card. I'll put a forced holoform disable in so it doesn't just poof.